bower.json

```diff
@@ -46,6 +47,17 @@
         "less/font-awesome.less",
         "./fonts/*"
       ]
+    },
+    "moment": {
```

Comment:
Hey @alexnoox 
In angular moment docs:
```
Usage
Include both moment.js and angular-moment.js in your application.
```
My input
This solution is out of the box and works by loading the different locales files we need in the project from the bower.json. The best part is that we can load all of them, or also only the ones we need. I specified id and ch to avoid having unnecessary extra files, but it can be extended anytime!
  "moment": {
       "main": [
         "moment.js",
         "locale/*.js"

Once it is loaded, by passing the locale we are using to moment in the run, or in the select language box for changing it.
```
moment.locale(TheLocaleWeAreUsing)
```
The language will be set globally and therefore propagated to every bit in the code where we are binding the result of moment. 
The default language is English, it does not need to be loaded. 
I tried with Chinese, french and Indonesian and seems to work perfectly fine, so it is very easy to extend.